### PR TITLE
Add the code to the readme.

### PR DESCRIPTION
**Pull request by Dee68**
better code the project.

### CODE REVIEW

**Comment by Dee68** [comment]
LGTM(Looks good to me)

**Comment by Dee68** [comment]
good

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # test_github
 
-## Coding the project
```
Comment (deewap) [inline README.md:3]: LGTM

Comment (deewap) [reply README.md:3]: Leave it at that.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
\ No newline at end of file
+## Coding
```
Comment (deewap) [inline README.md:3]: good

**Review by deewap — APPROVED** [review]

**Pull request merged**